apple/internal/apple_framework_import.bzl

```diff
@@ -92,6 +92,10 @@ def _apple_framework_import_impl(ctx):
     framework_dirs_set = depset(framework_groups.keys())
     objc_provider_fields = {}
 
+    transitive_sets = []
+    for dep in ctx.attr.deps:
+        transitive_sets.append(dep[AppleFrameworkImportInfo].framework_imports)
```

Comment:
`if hasattr(dep[AppleFrameworkImportInfo], "framework_imports"):` to skip the providers that don't have the field

Reply:
Since we're requiring this in the `deps` `providers` array I don't think we need to do this? Would you still like to in case that changes in the future?

Reply:
Ah this is for the framework_imports field, not the provider itself because of the change below. Done